subjects/scripting/pyFuncNames.reko/pySample_text.c

```diff
@@ -39,6 +39,10 @@ void initpySample()
 	//         Address: 0x10001140
 	//         File: None
 	//         Decompile: True
+	//     empty_procedure_loaded:
```

Comment:
@ptomin : is this a regression? I'm not certain of how to interpret it.

Reply:
Yes, it is. `empty_procedure_loaded` defined at https://github.com/uxmal/reko/blob/8500b5794287d1f63e228add20422d93a8d8f393/subjects/scripting/py_func_names.py#L68
which is later than `dump_procedures` https://github.com/uxmal/reko/blob/8500b5794287d1f63e228add20422d93a8d8f393/subjects/scripting/py_func_names.py#L57
So it should not be there